src/communicationmanager.ts

```diff
@@ -178,13 +178,13 @@ export class CommunicationManager {
         instrumentIp?: string,
         usb_unique_string?: string,
         filePath?: string
-    ): string {
-        let info = ""
+    ): [info: string, verified_name?: string] {
```

Comment:
Please add a Doc Comment to this function that describes the parameters and the return value. It is too important a function to just have a couple line comments.